config/default/manager_config_patch.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       - name: manager
         args:
         - "--config=controller_manager_config.yaml"
+        - "--zap-devel"
```

Comment:
Maybe we should have another kustomize folder for devel rules? Maybe we could even set the log level to 3. And add it to /docs/setup/install.md

Reply:
Devel combines couple  things  - encoder, log level and stack trace level. Although stack level seems wierd as devel flag doc suggests WARN level, while the stack level flag doesn't allow it so i guess it is also error, same as for default config. 

If you don't like the devel falg we could use explicit encode name - combined with other other flags it will have the same effect.  But I would suggest to go with human readable logs by default - I guess not all company-wide logs solutions are compatible with the zap json log format.

Reply:
It's not so much about the name of the flag, but encouraging the best practice (structured logging) in production.
Having a separate folder for the developer-friendly setup would be a good middle ground.

Reply:
Added a separate folder/manifest. PTAL.